shared_tasks/dynalab/README.md

```diff
@@ -89,16 +89,45 @@ based your model's specific dependencies.
 Once you've implemented the handler you'll need to test it locally.
 
 First install `dynalab` using instructions from their [repo](https://github.com/facebookresearch/dynalab#installation).
+### Project installation and dependencies 
+
+#### 1. Create & activate virtual venv 
+```angular2html
+python3 -m venv venv
+source venv/bin/activate
+```
+#### 2. Install Dynalab
+```angular2html
+git clone https://github.com/facebookresearch/dynalab.git
+cd dynalab
+pip install -e .
+```
+#### 3. Install fairseq & sentencepiece 
```

Comment:
I don't think you need that many sub header. Just keep one block of commands and insert comment inside if needed.